tools/BUILD

```diff
@@ -10,10 +10,10 @@ load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
 nodejs_binary(
     name = 'quicktype_runner',
     data = [
-        ":quicktype_runner.js",
+        "quicktype_runner.js",
     ],
-    node_modules = "//:quicktype_node_modules",
-    entry_point = "angular_devkit/tools/quicktype_runner.js",
+    node_modules = "@npm//quicktype-core",
```

Comment:
there should be a corresponding delete of the code that declared the quicktype_node_modules

Reply:
done